<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>encW</class>
 <widget class="QWidget" name="encW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>input</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select folder</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>output</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ENCODE WINDOW</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Caesar Cipher</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polybius square</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
